component/tests/golden/vshn/appcat/appcat/sli_exporter/90_slo_vshn_keycloak.yaml

```diff
@@ -0,0 +1,206 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  annotations: {}
+  labels:
+    name: vshn-keycloak
+  name: vshn-keycloak
+  namespace: appcat-slos
+spec:
+  groups:
+    - name: sloth-slo-sli-recordings-appcat-vshn-keycloak-uptime
+      rules:
+        - expr: |
+            (sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNKeycloak", ha="false", maintenance="false"}[5m]) or 0*rate(appcat_probes_seconds_count{service="VSHNKeycloak"}[5m])) by (service, namespace, name, organization, sla))
+            /
+            (sum(rate(appcat_probes_seconds_count{service="VSHNKeycloak", ha="false"}[5m])) by (service, namespace, name, organization, sla))
+          labels:
+            sloth_id: appcat-vshn-keycloak-uptime
+            sloth_service: appcat-vshn-keycloak
+            sloth_slo: uptime
```

Comment:
is sliprober ready for that?

Reply:
Will be implemented later, for now this is a noop.